### ISSUE TITLE
NTR: Thyroglobulin low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8338]
New term request Thyroglobulin low in blood (elthat_130430154337):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000244
- label: Decreased circulating thyroglobulin concentration
- parents: [Abnormal circulating thyroglobulin level (HP:0025483)](https://hpo.jax.org/app/browse/term/HP:0025483)
- pmid: 
- def: The concentration of thyroglobulin in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8338
- synonyms:
